### PR TITLE
Revoke org credentials

### PR DESCRIPTION
**Pull request by dmeijboom**
Revoke organisations credentials after updating organization details

### CODE REVIEW

File: domain/credentials/service.go [hunk 1]
```diff
@@ -46,28 +46,27 @@ func (s Service) ManageNutsOrgCredential(customer domain.Customer, shouldHaveCre
 		if customer.City == nil {
 			return fmt.Errorf("customer.City must be set for issuing a credential")
 		}
-		issue := true
-		for _, credential := range credentials {
-			if credential.Organization.Name == customer.Name && credential.Organization.City == *customer.City {
-				// Customer already has credential with given name/ & city, so we don't need to issue one.
-				issue = false
-				break
+
```
Comment (stevenvegt) [inline domain/credentials/service.go:49]: Indenting is weird?

File: domain/credentials/service.go [hunk 2]
```diff
@@ -46,28 +46,27 @@ func (s Service) ManageNutsOrgCredential(customer domain.Customer, shouldHaveCre
 		if customer.City == nil {
 			return fmt.Errorf("customer.City must be set for issuing a credential")
 		}
-		issue := true
-		for _, credential := range credentials {
-			if credential.Organization.Name == customer.Name && credential.Organization.City == *customer.City {
```
Comment (stevenvegt) [inline domain/credentials/service.go:51]: So you now always recreate the credentials, even if name and city are unchanged?

File: domain/credentials/service.go [hunk 3]
```diff
@@ -46,28 +46,27 @@ func (s Service) ManageNutsOrgCredential(customer domain.Customer, shouldHaveCre
...
+				return fmt.Errorf("unable to revoke NutsOrgCredentials for customer %d: %w", customer.Id, err)
 			}
 		}
-		if issue {
-			err = s.issueNutsOrgCredential(customer)
+
+		if err := s.issueNutsOrgCredential(customer); err != nil {
+			return fmt.Errorf("unable to manage NutsOrgCredential for customer %d: %w", customer.Id, err)
 		}
 	} else {
```
Comment (stevenvegt) [inline domain/credentials/service.go:59]: It looks you are always revoking in both the `shouldHaveCredential` is true or false. Perhaps take it out of the conditional and remove the else?

**Review by stevenvegt — CHANGES_REQUESTED** [review]
A few questions in the comments. And can you remove the package-lock.json from this PR?

File: domain/credentials/service.go (hunk 2 above)
Comment (dmeijboom) [reply domain/credentials/service.go:51]: Yes

File: domain/credentials/service.go (hunk 1 above)
Comment (dmeijboom) [reply domain/credentials/service.go:49]: It's not? Note that I'm running `gofmt` on every file.

**Comment by dmeijboom** [comment]
> package-lock.json from this PR?

Sure. But it's broken so I'll add it to another PR then

**Review by stevenvegt — APPROVED** [review]

**Pull request merged**